component/common/net/config.go

```diff
@@ -1,13 +1,13 @@
-// Package http contains a River serializable definition of the weaveworks weaveworks config in
-// https://github.com/weaveworks/common/blob/master/server/server.go#L62.
+// Package http contains a River serializable definition of the dskit dskit config in
```

Comment:
Haha, we can probably fix the duplicated word here:

```suggestion
// Package http contains a River serializable definition of the dskit config in
```